core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.component.html

```diff
@@ -57,7 +57,7 @@
           </span>
         </span>
         <div class="skill-list-options-container">
-          <p class="fas fa-ellipsis-v skill-edit-box-icon protractor-test-skill-edit-box" (click)="changeEditOptions(skill.id)"></p>
+          <p class="fas fa-ellipsis-v skill-edit-box-icon protractor-test-skill-edit-box" aria-label="Edit option for skill, <[skill.description]>" (click)="changeEditOptions(skill.id)"></p>
```

Comment:
@jimbyo We use `{{ }}` for interpolation in Angular 2+, right?

Reply:
Huh! it looks like we do according to this. https://github.com/oppia/oppia/wiki/Angular-Migration

Hm, but I see instances of <[]> throughout the code base. When are we supposed to use {{}} and <[]>

Reply:
We use `<[ ]>` for interpolation in ajs while `{{ }}` for Angular 2+.For instances where you have seen `<[ ]>` must have not been migrated. Here we would do something like `attr.aria-label = "Edit option for skill, {{ skill.description }}` .Note here `attr` is used for binding purposes.